hls4ml/templates/vivado/nnet_utils/nnet_common.h

```diff
@@ -24,6 +24,7 @@ namespace nnet {
 // Common type definitions
 enum io_type { io_parallel = 0, io_stream };
 enum strategy { latency, resource };
+enum class conv_implementation { linebuffer = 0, encoded = 1, pointwise = 2 };
```

Comment:
Same question as before

Reply:
pointwise used by default by latest commit. i think you can still override it by skipping to the optimizer (to be checked).